src/test/scala/com/target/TestHelpers.scala

```diff
@@ -0,0 +1,59 @@
+package com.target.data_validator
+
+import io.circe.Json
```

Comment:
will need to add `import com.target.data_validator.VarSubstitution`

Reply:
see prev comment.